src/components/ViewingsList/ViewingsListCardForm.tsx

```diff
@@ -295,16 +342,44 @@ const ViewingsListCardForm = ({
         invalid={invalidFields?.address3}
         invalidText={invalidFields?.address3}
       />
+
       <Input
-        id={`addressUrl-${id}`}
-        name="addressUrl"
-        defaultValue={addressUrl}
-        labelText={addressUrlLabel}
-        size="sm"
-        type="url"
-        invalid={invalidFields?.addressUrl}
-        invalidText={invalidFields?.addressUrl}
+        id={`emailOn-${id}`}
+        type="toggle"
+        labelText={emailToggleLabel}
+        size="md"
+        defaultChecked={emailOnState}
+        inline
+        invalid={invalidFields?.emailOn}
+        invalidText={invalidFields?.emailOn}
+        value={true}
+        name="emailOn"
+        onChange={() => setEmailOnState((oldState) => !oldState)}
       />
+      {!emailOnState ? <input type="hidden" name="emailOn" value={'false'} /> : null}
+
+      <div
+        className={classnames(`${baseClass}__preview-set`, { [`${baseClass}__preview-set--hidden`]: !emailOnState })}
+      >
+        <Input
+          id={`email-${id}`}
+          name="email"
+          defaultValue={email}
+          labelText={emailLabel}
+          size="sm"
+          invalid={invalidFields?.address2}
+          invalidText={invalidFields?.address2}
```

Comment:
We may want to apply the hidden attribute based off the `emailOnState` like we do for Preview that you can see on line 303
```js
          hidden={!emailOnState}
```

Reply:
Added.